api/core/core.proto

```diff
@@ -318,11 +318,8 @@ message ListEventsResponse {
 }
 
 message SyncFluxObjectRequest {
-    string         name        = 1;
-    string         namespace   = 2;
-    FluxObjectKind kind        = 3;
-    string         clusterName = 4;
-    bool           withSource  = 5;
+    repeated SuspendReqObj objects    = 1;
```

Comment:
Why does `SyncFluxObjectRequest` use `SuspendReqObj` too? I think it should have its own object type, even if fields are the same? Or if it is necessary to share this object type between both `ToggleSuspendResourceRequest` and `SyncFluxObjectRequest` the type name should be more generic? @ozamosi , what do you think about it?

Reply:
Honestly I just didn't come up with a name I liked - SyncSuspendReqObj is just....bleh you know? If you have a good name it's an easy change

Reply:
So. I looked what we have.
* This is a `FluxObjectRef`, except with `clusterName`. `FluxObjectRef` is used together with a `clusterName` field in both kustomizations and helmreleases. So it shouldn't really be added there, instead the pattern with that one is to have one field with the object ref and a different field with the clusterName.
* That in turn is basically the same as `ObjectRef` except with the kind as an enum instead of a string. `ObjectRef` is only used by events, which do not specify cluster at all, which is incorrect.

`listObjects` spirits be willing, `FluxObjectRef` won't need to be a thing for long, so that duplication will be got rid of.

`ObjectRef` needs to grow a `clusterName` regardless of what we do here so events work properly in multi cluster.

I don't like the `FluxObjectKind` enum because it's not extensible, so I'd rather not use it.

So, this should just be an (improved version of) `ObjectRef`. Done, right?

However, the enummy bits are quite ingrained in the suspend/sync code, so that would grow the scope of this ticket quite a bit. So, no, out of scope.

Honestly, let's call it `ClusteredObjectRef` and put it next to the other object refs, thereby maximising the chance that it will nerd snipe someone into merging them all.

Reply:
Great work Robin thanks for looking into it. Will implement your change shortly

Reply:
@ozamosi great breakdown, Robin. ⭐